doc/host-setup.md

```diff
@@ -6,6 +6,22 @@ following options to load DRBD:
 
 [DRBD]: https://github.com/LINBIT/drbd/
 
+## Distro specific modifications
+
+Some distros need modifications made to them before Piraeus will work properly, primarily due to package version mismatches.
+
+### Ubuntu 20.04
+
+In Ubuntu 20.04, the multipath daemon opens up DRBD devices if it is not configured which prevents Piraeus from working properly.
+To configure the multipath daemon, put the following into the file `/etc/multipath/conf.d/drbd.conf`. You will need to create the
+directory `/etc/multipath/conf.d` on new installations of Ubuntu 20.04.
+
+```
+blacklist {
+        devnode "^drbd[0-9]+"
+}
+```
```

Comment:
I'd rephrase this section a bit: the problem with `multipathd` doesn't only exist on ubuntu 20.04, but basically in any default multipath installation. The difference is that Ubuntu 20.04 has `multipathd` installed and enabled by default.

So in essence I'd change this section to say "If you are using `multipathd`..." and add that Ubuntu 20.04 enables that by default as a note.

Reply:
I just adjusted the section to be labelled "Software compatibility" instead of making it distro specific and I updated the multipath section to account for the changes that you requested. It should be more general now for all multipath installations.